sympy/polys/rings.py

```diff
@@ -1953,9 +1955,10 @@ def imul_num(p, c):
             p[exp] *= c
         return p
 
-    def content(f):
+    def content(f, domain=None):
         """Returns GCD of polynomial's coefficients. """
-        domain = f.ring.domain
+        if domain is None:
+            domain = f.ring.domain
```

Comment:
What is the purpose of the `domain` argument here?

I don't like that argument being there on a public method because it should not be valid to pass any value other than `f.ring.domain`.